iOSClient/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -588,7 +588,7 @@
 "_password_pdf_error_"          = "Wrong password";
 "_error_download_photobrowser_" = "Error: Unable to download photo";
 "_good_morning_"                = "Good morning";
-"_good_noon_"                   = "Good noon";
+"_good_noon_"                   = "Good day";
```

Comment:
Why "Good day" if the text string appears exactly at 12 o'clock?
https://github.com/nextcloud/ios/blob/17443651c8b978eef74b046241a72ebd936d5476/Widget/Nextcloud/NextcloudData.swift#L62